etl/collections/explorer.py

```diff
@@ -31,17 +31,9 @@ class Explorer(Collection):
     definitions: Optional[Definitions] = None
     avoid_duplicate_hack: bool = False
 
-    # Internal use. For save() method.
-    _catalog_path: Optional[str] = None
-
-    @property
-    def catalog_path(self) -> Optional[str]:
-        return self._catalog_path
-
-    @catalog_path.setter
-    def catalog_path(self, value: str) -> None:
-        assert "#" in value, "Catalog path should be in the format `path#name`."
-        self._catalog_path = value
+    def __post_init__(self):
+        # Internal use
+        self._collection_type = "explorer"
```

Comment:
Nit - does it make a difference if you set it in `__post_init__` or if you define it as a constant on the class itself? i.e.

```
class Explorer:
    ...
    avoid_duplicate_hack: bool = False
    _collection_type = "explorer"
```